### ISSUE TITLE
I tried to deploy this on Qovery but keep getting a Failure error.

### ISSUE THREAD

**Issue by bnichs5** [rhildred/docker-wordpress#1]
I tried to deploy this on Qovery but keep getting a Failure error. Any ideas why that may be happening?

**Comment by rhildred** [comment]
You need to edit the qovery.yml file again after you bring up the project. I think it is the public part that you are missing????

**Comment by bnichs5** [comment]
Thanks that helped. It worked for a couple days but now I am receiving a different error. 

This site can’t provide a secure connection
XXXXXX-XXXXXXX-XXXXX.qovery.io sent an invalid response.

Try running Windows Network Diagnostics.
ERR_SSL_PROTOCOL_ERROR

Any ideas?

**Comment by rhildred** [comment]
Yes, that all looks good to me. The error ERR_SSL_PROTOCOL_ERROR looks like it might be certificate related. Have you tried qovery chat support. They were really helpful with me.
